src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup.mdx

```diff
@@ -125,21 +136,35 @@ Before you go to the external OTLP exporter documentation, consult the table bel
 In Node.js, the [opentelemetry-collector-exporter-grpc](https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-grpc) library requires additional options to enable TLS.
```

Comment:
this seems like a random place for this tip. something to think about where the best place might be. Is it weird that Node.js isn't one of the languages mentioned above but is mentioned here? not sure.